storage/campaignschedules.go

```diff
@@ -24,3 +25,14 @@ func (db *store) CreateCampaignSchedule(c *entities.CampaignSchedule) error {
 func (db *store) DeleteCampaignSchedule(campaignID int64) error {
 	return db.Where("campaign_id = ?", campaignID).Delete(entities.CampaignSchedule{}).Error
 }
+
+// GetScheduledCampaigns returns all scheduled campaigns < time
+func (db *store) GetScheduledCampaigns(time time.Time) ([]entities.CampaignSchedule, error) {
+	var campaignsSchedule []entities.CampaignSchedule
+	err := db.Joins("LEFT JOIN campaigns ON campaigns.id = campaign_schedules.campaign_id").
```

Comment:
```suggestion
	err := db.Joins("JOIN campaigns ON campaigns.id = campaign_schedules.campaign_id").
```

We should only select records with matching values here. The default join should do the trick.